tests/debuginfo/collapse-debuginfo-in-non-collapse-macro.rs

```diff
@@ -0,0 +1,120 @@
+// ignore-lldb
+#![feature(collapse_debuginfo)]
+
+// Test that statement, skipped by macros, is correctly processed in debuginfo
```

Comment:
These comments in the tests could be a bit more descriptive too - these debuginfo tests are a bit tricky to read at the best of times and a comment describing the specific observation this test is hoping to make would be helpful.

Reply:
Comments improved.